app/src/views/Home.vue

```diff
@@ -54,7 +54,7 @@
                     <v-col cols="8" sm="9" md="4" class="pl-10">
                       <radius />
                     </v-col>
-                    <v-col cols="4" sm="3" md="2" class="pl-10 py-1">
+                    <v-col cols="4" sm="3" md="2" class="pl-12 py-1">
```

Comment:
On small screens this works fine.
![grafik](https://user-images.githubusercontent.com/64203299/85924247-3eb46580-b891-11ea-9a68-6917cb39a635.png)
But on larger screens it does not work.
![grafik](https://user-images.githubusercontent.com/64203299/85924258-5c81ca80-b891-11ea-8bfd-82b614738cce.png)

I think the pl class is not the best practise.
@jschurse Do you maybe know a better practise?

Reply:
He suggested one above. Trying it out since ive pretty much ran out of options